code/modules/unit_tests/breath.dm

```diff
@@ -1,73 +1,96 @@
-/// Tests to make sure humans can breath in normal situations
+/// Tests to make sure humans can breath in normal situations.
+/// Ensures algorithmic correctness of the "breathe()" and "toggle_internals()" procs.
 /// Built to prevent regression on an issue surrounding QUANTIZE() and BREATH_VOLUME
 /// See the comment on BREATH_VOLUME for more details
 /datum/unit_test/breath_sanity
 
-/datum/unit_test/breath_sanity/Run()
-	var/mob/living/carbon/human/lab_rat = allocate(/mob/living/carbon/human/consistent)
-	var/obj/item/clothing/mask/breath/tube = allocate(/obj/item/clothing/mask/breath)
-	var/obj/item/tank/internals/emergency_oxygen/source = allocate(/obj/item/tank/internals/emergency_oxygen)
-
-	lab_rat.equip_to_slot_if_possible(tube, ITEM_SLOT_MASK)
+/// Equips the given Human with a new instance of the given tank type and a breathing mask.
+/// Returns the new equipped tank.
+/datum/unit_test/proc/equip_labrat_internals(mob/living/carbon/human/lab_rat, tank_type)
```

Comment:
I rather like the idea of using #2 since it's very minimal complexity, and allows for future extension in other tests that add their own utility procs.